### PR TITLE
Modern layout

### PR DESCRIPTION
**Pull request by Ma1ithorne**
Hey, this version has the addition of the layout file BlizzModern.lua.
Blizz Modern sample2

### CODE REVIEW

**Comment by Sammers21** [comment]
@Ma1ithorne is the Pull request is ready to be reviewed? I'm trying to test the changes and i cant see the layout available for selection atm: 

image

**Comment by Ma1ithorne** [comment]
@Sammers21 Hey, you are correct - I forgot to add that to the .xml file here, fixed it now.

**Comment by jomoroon** [comment]
wow this is a nice skin, any chance of adding as an option as part of your default addon sammers?

**Comment by Sammers21** [comment]
@Ma1ithorne i'm still having the same issue:
image
The layout is not available for selection for me. Can you fix that? How do you making it work for yourself?

**Comment by Ma1ithorne** [comment]
@Sammers21 That is strange, I re-downloaded this fork, moved the main addOn elsewhere temporarily, removed the '-master' at the end and it shows up for me. Do you have multiple versions in the addOn folder at once?
image

image

**Comment by Sammers21** [comment]
@Ma1ithorne yes, i did exactly what you said. I will try again soon. Maybe i did something wrong.

**Comment by Sammers21** [comment]
@Ma1ithorne i have tried it again and its working fine for me.

File: sArena.xml [hunk 1]
```diff
@@ -252,4 +253,4 @@
             <OnLoad method="OnLoad"/>
         </Scripts>
     </Frame>
-</Ui>
\ No newline at end of file
+</Ui>
```
Comment (Sammers21) [inline sArena.xml:256]: Can you make it so this like is not changed?

File: Layouts/BlizzModern.lua [hunk 2]
```diff
@@ -0,0 +1,263 @@
+local layoutName = "Blizz Modern"
+local layout = {}
+layout.name = "|cff00b4ffBlizz|r Modern by |cffFF7C0AMalithorn|r"
```
Comment (Sammers21) [inline Layouts/BlizzModern.lua:3]: Can we change this to just `Blizz Modern`? I will mention your contribution in README and on the Curseforge description. Is that okay?

**Review by Sammers21 — CHANGES_REQUESTED** [review]

File: Layouts/BlizzModern.lua (hunk 2 above)
Comment (Sammers21) [reply Layouts/BlizzModern.lua:3]: Or you want your name on the layout can just call it `Malithorn` without `Blizzard Modern`

**Comment by Sammers21** [comment]
@Ma1ithorne do the changes are we are good to merge/release

**Comment by Sammers21** [comment]
image
This is how the layout look in shuffle. Can you fix it?

**Comment by Ma1ithorne** [comment]
> image This is how the layout look in shuffle. Can you fix it? @Ma1ithorne

Hey, fixed the default trinket/racial position.

File: sArena.xml (hunk 1 above)
Comment (Ma1ithorne) [reply sArena.xml]: > Can you make it so this line is not changed?

Hey, this should be fixed now.

**Review by Sammers21 — APPROVED** [review]
Looks good to me

**Review by Sammers21 — APPROVED** [review]
Looks good to me

**Comment by Sammers21** [comment]
@Ma1ithorne the changes has been released to curse forge. Thank you for the contribution and good job!

**Pull request merged**